contracts/test/staking/single-asset-staking.js

```diff
@@ -121,6 +121,46 @@ describe("Single Asset Staking", function () {
     );
   });
 
+
+  it("Stake using WithSender with correct rewards", async () => {
+    const { ogn, anna, ognStaking} = await loadFixture(defaultFixture);
+
+    const annaStartBalance = await ogn.balanceOf(anna.address);
+
+    const numStakeAmount = 1;
+    const stakeAmount = ognUnits(numStakeAmount.toString());
+    // 0.085 is the default reward for three months
+    const expectedReward = ognUnits((numStakeAmount * 0.085).toString());
+
+    await expect(ognStaking.connect(anna).stakeWithSender(anna.address, stakeAmount, threeMonth))
+      .to.be.revertedWith("Only token contract can make this call");
+
+    // from franck
```

Comment:
nit: cleanup since not really relevant to leave my name, especially since crazybuster found a better way:)

Reply:
cleaned up